screwdriver.yaml

```diff
@@ -1,5 +1,7 @@
 shared:
-    image: centos
+    image: golang:1.12
```

Comment:
Not using latest version is there any reason?
> now latest version is 1.13

Reply:
I think use latest is also fine.

Reply:
Updated image tag and my local golang environment 🙏